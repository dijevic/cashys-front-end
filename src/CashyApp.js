import { AppRouter } from "./Routes/AppRouter"

const CashyApp = () => {
    return (
        <AppRouter />
    )
}

export default CashyApp