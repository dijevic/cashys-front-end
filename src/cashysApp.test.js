import { shallow } from "enzyme/build"
import CashyApp from "./CashyApp"

describe('test app in general', () => {

    test('test app matching snapshot', () => {
        const wrapper = shallow(<CashyApp />)
        expect(wrapper).toMatchSnapshot()
    })
})