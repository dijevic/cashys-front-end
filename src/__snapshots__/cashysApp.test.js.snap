// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test app in general test app matching snapshot 1`] = `<AppRouter />`;
